package.json

```diff
@@ -24,8 +24,6 @@
     "@testing-library/dom": "^10.1.0",
```

Comment:
I suspect we might be able to drop `@babel/preset-env` - I think we were only using it to make Jest play nicely with ESM. If so, we can also drop the babel.config.json in the root folder.

Reply:
Removed now